src/integrationTest/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/resource/UpdateInductionTest.kt

```diff
@@ -839,16 +821,16 @@ class UpdateInductionTest : IntegrationTestBase() {
     reasonToNotGetWork: Set<ReasonNotToWork>? = originalInduction.reasonToNotGetWork,
     reasonToNotGetWorkOther: String? = originalInduction.reasonToNotGetWorkOther,
     workExperience: UpdatePreviousWorkRequest? = aValidUpdatePreviousWorkRequest(
-      id = originalInduction.workExperience!!.id!!,
+      id = originalInduction.workExperience?.id,
     ),
     skillsAndInterests: UpdateSkillsAndInterestsRequest? = aValidUpdateSkillsAndInterestsRequest(
-      id = originalInduction.workExperience!!.id!!,
+      id = originalInduction.skillsAndInterests?.id,
     ),
     qualificationsAndTraining: UpdateEducationAndQualificationsRequest? = aValidUpdateEducationAndQualificationsRequest(
-      id = originalInduction.workExperience!!.id!!,
+      id = originalInduction.qualificationsAndTraining?.id,
     ),
     inPrisonInterests: UpdatePrisonWorkAndEducationRequest? = aValidUpdatePrisonWorkAndEducationRequest(
-      id = originalInduction.workExperience!!.id!!,
+      id = originalInduction.inPrisonInterests?.id,
```

Comment:
My bad. I'm intrigued why all the tests still passed, but I can only assume that the existing ID didn't matter (i.e. the exisiting object was overwritten regardless). Something for tomorrow.